docs/libssh2_userauth_publickey_fromfile_ex.3

```diff
@@ -2,7 +2,7 @@
 .\" SPDX-License-Identifier: BSD-3-Clause
 .TH libssh2_userauth_publickey_fromfile_ex 3 "1 Jun 2007" "libssh2 0.15" "libssh2"
 .SH NAME
-libssh2_userauth_publickey_fromfile_ex - authenticate a session with a public key, read from a file
+libssh2_userauth_publickey_fromfile_ex - authenticate a session based on local public key cryptography
```

Comment:
This is true, but isn't this tagline true for all `libssh2_userauth_publickey*` functions?
The 'read from a file' differenciated this from the rest of similar calls, thus having this 
it in the title seems useful. Also, to me, 'local public key cryptography' rings 
unfamiliar; the common wording being 'public key authentication'.

Reply:
I reworded the line to be more clear about using a local file